tox.ini

```diff
@@ -56,7 +56,7 @@ description = invoke sphinx-build to build the HTML docs
 extras = docs
 commands =
     pip freeze
-    sphinx-build -W -b html . _build/html
```

Comment:
Why this change? Ideally, we fix whatever causes the warnings!

Was it this warning from the log that caused the problem?
```
/home/docs/checkouts/readthedocs.org/user_builds/pyerfa/envs/100/lib/python3.7/site-packages/sphinx_astropy/conf/v1.py:227: UserWarning: matplotlib's plot_directive could not be imported. Inline plots will not be included in the output
```
This sounds like a bug in `sphinx-astropy`, though if takes setting `matplotlib` as a docs dependency to remove the warning and get the docs to build with `-W`, then perhaps that is worth it. Pinging @astrofrog...

Reply:
It seems that this is no longer an issue ... even if it is not clear to me why

Reply:
??? Well, we'll take it!